draft-ietf-quic-multipath.md

```diff
@@ -186,6 +185,16 @@ capitals, as shown here.
 We assume that the reader is familiar with the terminology used in
 {{QUIC-TRANSPORT}}. When this document uses the term "path", it refers
 to the notion of "network path" used in {{QUIC-TRANSPORT}}.
+In addition, we define the following term:
+
+- Path Identifier (Path ID): An identifier that is used to identify 
+  a path in a QUIC connection at an endpoint. Path Identifier is used 
+  in multipath control frames (etc. PATH_ABANDON frame) to identify a path. 
+  Connection IDs are issued per path ID. When endpoints address a path in 
+  multipath control frames, it refers to the Path Identifier related to
+  the destination Connection ID used for sending packets on that 
+  particular path. 
```

Comment:
When it comes to path ID definition, the logic problem in the statement "Path Identifier (Path ID): An identifier that is used to identify a path...." lies in its redundancy and circular definition. It essentially defines a term using the same concept embedded within the term itself, which doesn't provide any additional information or clarity.

Reply:
Actually think this is fine, because that's what it is and identifier to identify the concept of a path. I think what is not explained here is what's the concept of a path we are using or how a path is defined. However, that is defined later and I think becomes clear in the doc.

We could do a minor editorial change and say "This identifier defines a path in a QUIC connection" but I don't think it makes a real different.

@yfmascgy do you have a proposal how to phrase it better?

Reply:
I think one can clarify that a path is defined as unidirectional forwarding path from A to B. Where the path ID has a set of Destination CIDs that identify packets belonging to the ID. An endpoint will bind its local 5-tuple to the PATH ID when first sending or receiving on the ID.

Reply:
> I think one can clarify that a path is defined as unidirectional forwarding path from A to B. Where the path ID has a set of Destination CIDs that identify packets belonging to the ID. An endpoint will bind its local 5-tuple to the PATH ID when first sending or receiving on the ID.

I can see the difficulty in path definition here, especially when continuity of a path is involved. I am not sure defining a path as "unidirectional forwarding path from A to B" is a good definition because it does provide a set of criteria that uniquely distinguishes a path from the others.